apps/ui/src/app/[locale]/dev/showcase/components/strapiComponents/MockedStrapiHero.tsx

```
@@ -0,0 +1,28 @@
+import type { Data } from "@repo/strapi-types"
```

ESLint

[error] Resolve error: Tsconfig not found `@repo/typescript-config/base.json`
    at unrsResolve (/home/jailuser/git/node_modules/.pnpm/eslint-import-resolver-typescript@4.4.4_eslint-plugin-import-x@4.16.2_@typescript-eslin_ba92e77c93bacfed649305755a416e03/node_modules/eslint-import-resolver-typescript/lib/index.cjs:262:59)
    at Object.resolve (/home/jailuser/git/node_modules/.pnpm/eslint-import-resolver-typescript@4.4.4_eslint-plugin-import-x@4.16.2_@typescript-eslin_ba92e77c93bacfed649305755a416e03/node_modules/eslint-import-resolver-typescript/lib/index.cjs:333:19)
    at resolveWithLegacyResolver (file:///home/jailuser/git/node_modules/.pnpm/eslint-plugin-import-x@4.16.2_@typescript-eslint+utils@8.59.2_eslint@9.39.4_typescript@6.0.3__eslint@9.39.4/node_modules/eslint-plugin-import-x/lib/utils/legacy-resolver-settings.js:8:25)
    at file:///home/jailuser/git/node_modules/.pnpm/eslint-plugin-import-x@4.16.2_@typescript-eslint+utils@8.59.2_eslint@9.39.4_typescript@6.0.3__eslint@9.39.4/node_modules/eslint-plugin-import-x/lib/utils/resolve.js:203:64
    at setRuleContext (/home/jailuser/git/node_modules/.pnpm/eslint-import-context@0.1.9_unrs-resolver@1.11.1/node_modules/eslint-import-context/lib/index.js:23:20)
    at fullResolve (file:///home/jailuser/git/node_modules/.pnpm/eslint-plugin-import-x@4.16.2_@typescript-eslint+utils@8.59.2_eslint@9.39.4_typescript@6.0.3__eslint@9.39.4/node_modules/eslint-plugin-import-x/lib/utils/resolve.js:203:34)
    at relative (file:///home/jailuser/git/node_modules/.pnpm/eslint-plugin-import-x@4.16.2_@typescript-eslint+utils@8.59.2_eslint@9.39.4_typescript@6.0.3__eslint@9.39.4/node_modules/eslint-plugin-import-x/lib/utils/resolve.js:215:12)
    at resolve (file:///home/jailuser/git/node_modules/.pnpm/eslint-plugin-import-x@4.16.2_@typescript-eslint+utils@8.59.2_eslint@9.39.4_typescript@6.0.3__eslint@9.39.4/node_modules/eslint-plugin-import-x/lib/utils/resolve.js:220:16)
    at ExportMap.get (file:///home/jailuser/git/node_modules/.pnpm/eslint-plugin-import-x@4.16.2_@typescript-eslint+utils@8.59.2_eslint@9.39.4_typescript@6.0.3__eslint@9.39.4/node_modules/eslint-plugin-import-x/lib/utils/export-map.js:88:22)
    at processBodyStatement (file:///home/jailuser/git/node_modules/.pnpm/eslint-plugin-import-x@4.16.2_@typescript-eslint+utils@8.59.2_eslint@9.39.4_typescript@6.0.3__eslint@9.39.4/node_modules/eslint-plugin-import-x/lib/rules/namespace.js:9:31)
    at Program (file:///home/jailuser/git/node_modules/.pnpm/eslint-plugin-import-x@4.16.2_@typescript-eslint+utils@8.59.2_eslint@9.39.4_typescript@6.0.3__eslint@9.39.4/node_modules/eslint-plugin-import-x/lib/rules/namespace.js:100:21)
    at ruleErrorHandler (/home/jailuser/git/node_modules/.pnpm/eslint@9.39.4/node_modules/eslint/lib/linter/linter.js:1173:33)
    at /home/jailuser/git/node_modules/.pnpm/eslint@9.39.4/node_modules/eslint/lib/linter/source-code-visitor.js:76:46
    at Array.forEach (<anonymous>)
    at SourceCodeVisitor.callSync (/home/jailuser/git/node_modules/.pnpm/eslint@9.39.4/node_modules/eslint/lib/linter/source-code-visitor.js:76:30)
    at /home/jailuser/git/node_modules/.pnpm/eslint@9.39.4/node_modules/eslint/lib/linter/source-code-traverser.js:291:18
    at Array.forEach (<anonymous>)
    at SourceCodeTraverser.traverseSync (/home/jailuser/git/node_modules/.pnpm/eslint@9.39.4/node_modules/eslint/lib/linter/source-code-traverser.js:290:10)
    at runRules (/home/jailuser/git/node_modules/.pnpm/eslint@9.39.4/node_modules/eslint/lib/linter/linter.js:1214:12)
    at `#flatVerifyWithoutProcessors` (/home/jailuser/git/node_modules/.pnpm/eslint@9.39.4/node_modules/eslint/lib/linter/linter.js:2115:4)
    at Linter._verifyWithFlatConfigArrayAndWithoutProcessors (/home/jailuser/git/node_modules/.pnpm/eslint@9.39.4/node_modules/eslint/lib/linter/linter.js:2203:43)
    at Linter._verifyWithFlatConfigArray (/home/jailuser/git/node_modules/.pnpm/eslint@9.39.4/node_modules/eslint/lib/linter/linter.js:2306:15)
    at Linter.verify (/home/jailuser/git/node_modules/.pnpm/eslint@9.39.4/node_modules/eslint/lib/linter/linter.js:1677:10)
    at Linter.verifyAndFix (/home/jailuser/git/node_modules/.pnpm/eslint@9.39.4/node_modules/eslint/lib/linter/linter.js:2571:20)
    at verifyText (/home/jailuser/git/node_modules/.pnpm/eslint@9.39.4/node_modules/eslint/lib/eslint/eslint-helpers.js:1180:45)
    at readAndVerifyFile (/home/jailuser/git/node_modules/.pnpm/eslint@9.39.4/node_modules/eslint/lib/eslint/eslint-helpers.js:1321:10)

(import-x/namespace)